manifests/service.pp

```diff
@@ -40,17 +40,16 @@
   if $service_manage {
     case $::osfamily {
       'OpenBSD': {
-        service { 'nginx':
+        service { $service_name:
           ensure     => $service_ensure_real,
-          name       => $service_name,
           enable     => $service_enable,
           flags      => $service_flags,
           hasstatus  => true,
           hasrestart => true,
         }
       }
       default: {
-        service { 'nginx':
+        service { $service_name:
           ensure     => $service_ensure_real,
           name       => $service_name,
```

Comment:
Oh, and this line can be removed too

Reply:
Oops.  https://github.com/voxpupuli/puppet-nginx/pull/1159/commits/ad41e15349a61d357d5c41ecb795385f148b3555